comp/core/tagger/impl-dual/dual.go

```diff
@@ -46,7 +46,6 @@ func NewComponent(req Requires) (Provides, error) {
 			Config:    req.Config,
 			Log:       req.Log,
 			Telemetry: req.Telemetry,
-			Wmeta:     req.Wmeta,
```

Comment:
Since we no longer pass the workloadmeta component to the remote tagger, we can revert back the changes we made in the previous PR https://github.com/DataDog/datadog-agent/pull/30584/files#diff-f30d7d82577ab70305ba962ed703e91950b9fb1161df8a57ce0c98c9ed74187eR68 for the local tagger implementation? 

We change the dependecy of workloadmeta to be optional. For the local tagger that is no longer the case 😄